### ISSUE TITLE
Add information on how to cite maintainers in BEP papers

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/bids-extensions#13]
https://docs.google.com/document/d/1yQdYOw7WMpkcFgT4tNalOjKuH34nhmRqg4luI4V9uhE/edit#heading=h.rjf9c9gqie4f

**Comment by sappelhoff** [comment]
agreed that this would be nice, I'll take some time these days to go through the gdoc as well and check whether I can resolve some WIP items there.
